src/components/dateTimePicker/index.tsx

```diff
@@ -275,40 +248,65 @@ class DateTimePicker extends Component<DateTimePickerPropsInternal, DateTimePick
         themeVariant={themeVariant}
       />
     );
-  }
-
-  render() {
-    // @ts-expect-error
-    const textInputProps = TextField.extractOwnProps(this.props);
-    const {renderInput, editable} = this.props;
+  }, [
+    mode,
+    value,
+    handleChange,
+    minimumDate,
+    maximumDate,
+    locale,
+    is24Hour,
+    minuteInterval,
+    timeZoneOffsetInMinutes,
+    themeVariant
+  ]);
 
+  const renderIOSExpandableOverlay = () => {
```

Comment:
Why not wrap with useCallback? It is passed as a prop...

Reply:
Cause we're not passing the callback but its value

Reply:
oh I missed the ()...